src/main/java/seedu/address/model/status/Status.java

```diff
@@ -4,16 +4,14 @@
  * Enum for the possible statuses an internship application can have.
  */
 public enum Status {
-    ACTIVE, INACTIVE, PLAN_TO_APPLY, APPLICATION_DONE, INTERVIEW_SCHEDULED, INTERVIEW_DONE, ACCEPTED, REJECTED;
+    WISHLIST, APPLIED, INTERVIEW, OFFERED, REJECTED;
```

Comment:
Regarding the INACTIVE status -- where if the (current date - application date) >= 2 months -- then we will automatically set the internship application to INACTIVE (cuz ghosted by company) to prevent cluttering of the application pool. 

Do you think it should still be kept? Or should we leave that out for now, work on this first, and think about the INACTIVE status in the future as an extension?

Reply:
Hmm, does this only apply to all status type or only like for Applied/Interview Statuses?
I think we can discuss this and if need be, implement for v1.4

Oh and the Parser works fine no worries! No need any changes, it passed all the test cases

Reply:
**"Does this only apply to all status type or only like for Applied/Interview Statuses?"**

Hmm yeah you brought up a good point about this. I guess it can be applied to WISHLIST as well, since if you don't apply by that specified date, we can set the status to INACTIVE(?). 

But these are all design considerations, so I agree that we can leave it v1.4 and discuss when we get there. 

**"Oh and the Parser works fine no worries! No need any changes, it passed all the test cases"**

Oh okay, awesome! Everything is good. I'll merge this PR now.